src/lib/moderation.ts

```diff
@@ -60,10 +61,9 @@ export function isAppLabeler(
     | AppBskyLabelerDefs.LabelerView
     | AppBskyLabelerDefs.LabelerViewDetailed,
 ): boolean {
-  if (typeof labeler === 'string') {
-    return BskyAgent.appLabelers.includes(labeler)
-  }
-  return BskyAgent.appLabelers.includes(labeler.creator.did)
+  const did = typeof labeler === 'string' ? labeler : labeler.creator.did
+  // assertion here that `did` starts with `did:`
+  return did.startsWith('did:') && BskyAgent.appLabelers.includes(did as Did)
```

Comment:
This is maybe a little weird, but this `includes` has a type of `did:${string}` now. Feels maybe slight unnecessary - and it's equally weird imo that, i.e. `ProfileView`'s `creator.did` wouldn't be typed as `Did` if this is what we want to do.

Reply:
@devinivy for context

Reply:
I agree, it's probably worth loosening those types to match the types we get out of lexicon for DIDs.  If we want to use the `Did` type internally to the package, I see no issue with that— just a little funky in the public API since we don't use this type for DIDs elsewhere.  @matthieusieben what do you think of that?

Reply:
I changed the type here so this shouldn't be needed anymore